### PR TITLE
Simplify 07-validation

### PR DESCRIPTION
**Pull request by jamesramsay**
- More clearly defined puzzle task
- Simpler set of attributes to validate

### CODE REVIEW

File: 07-validation/lib/index.coffee [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
 exports.validate = (data) ->
 
   # Very naive checks - change them!
   if data.id < 0 then return false
 
-  if data.fees.base <= 0 then return false
-  if data.fees.custom <= 0 then return false
-
-  if !data.terms then return false
+  if !data.name == 'John Doe' then return false
```
Comment (runk) [inline 07-validation/lib/index.coffee]: if data.name isnt 'John Doe' then return false

Comment (jamesramsay) [inline 07-validation/lib/index.coffee]: :+1: much nicer.

**Comment by runk** [comment]
:rocket:

**Comment by runk** [comment]
Nice stuff @jamesramsay

**Pull request merged**